src/layouts/sidebar-layout/sidebar-layout.scss

```diff
@@ -1,25 +1,76 @@
 @import '../../assets/colors';
 @import '../../assets/styles';
 
+// screen height without header
 $height-without-header: calc(100vh - #{$header-height});
-$header-and-padding: $header-height + $content-padding;
 
-// height without header and padding adjustments
-$content-height: calc(100vh - #{$header-and-padding});
+// screen height without header or content padding
+$content-height: calc(100vh - #{$header-height + $content-padding});
+
+// only modify the <Header /> that has this class (which should only be this layout)
+.sidebar-layout-header {
+  .c-header-content {
+    .c-header-title {
+      @media screen and (max-width: $medium-screen) {
+        flex: 1;
+      }
+    }
+
+    .c-header-title {
+      @media screen and (max-width: $medium-screen) {
+        flex: 1;
+        padding: unset;
+      }
+    }
+
+    .c-account-buttons {
+      @media screen and (max-width: $medium-screen) {
+        flex: 1;
+      }
+    }
+  }
+}
+
+.sidebar-layout-sidebar {
+  @media screen and (max-width: $large-screen) {
+    padding-left: unset;
+    width: $sidebar-width / 1.15;
+  }
+
+  @media screen and (max-width: $medium-screen) {
+    display: none;
+  }
+}
 
 .sidebar-layout-page-wrap {
   height: $height-without-header;
   margin-left: $sidebar-width;
   padding-top: $header-height;
 
+  @media screen and (max-width: $large-screen) {
+    margin-left: ($sidebar-width / 1.15);
+  }
+
+  @media screen and (max-width: $medium-screen) {
+    margin-left: 0;
+  }
+
   > .sidebar-layout-content {
     // create a stacking context so all internal z-indexes apply only between sibling elements
-    position: relative;
-    z-index: 0;
+    isolation: isolate;
```

Comment:
spread the knowledge!!

to be pedantic though, it's not fully equivalent. with this, absolute children will be positioned to probably the app root instead of the content.

but unless we need that, this is probably the better choice. docs say this is THE way to create a new stacking context without any other side effects.

Reply:
Oh, that makes sense. Dope!